target/linux/bcm27xx/image/Makefile

```diff
@@ -104,14 +109,20 @@ define Device/rpi-2
 	bcm2710-rpi-3-b bcm2710-rpi-3-b-plus \
 	bcm2711-rpi-4-b bcm2711-rpi-400 \
 	bcm2710-rpi-cm3 bcm2711-rpi-cm4 \
-	bcm2710-rpi-zero-2
+	bcm2710-rpi-zero-2 bcm2710-rpi-zero-2-w
   SUPPORTED_DEVICES := \
-	rpi-2-b rpi-3-b rpi-3-b-plus rpi-cm rpi-zero-2 \
+	rpi-2-b \
+	rpi-3-a-plus \
+	rpi-3-b rpi-3-b-plus \
+	rpi-4-b rpi-400 \
+	rpi-cm \
```

Comment:
I don't think that `rpi-cm` is correct here.
Maybe it was copy&pasted by mistake?

Reply:
@Noltari 
rpi-cm doesn't look right to me either but it was already there.
I went in to add rpi-3-a-plus to the list and in the process noticed that rpi-4-b and rpi-400 were also missing in the SUPPORTED_DEVICES so I added those too. It was starting to get hard to read so I split up the line (replaced line 109).
Along the same lines, raspberrypi,3-compute-module is present in SUPPORTED_DEVICES.
Is it safe to remove rpi-cm and raspberrypi,3-compute-module ?